packages/plugin-mongo/src/database.ts

```diff
@@ -43,4 +48,16 @@ export default class MongoDatabase {
   stop() {
     return this.client.close()
   }
+
+  connectionStringFromConfig() {
+    let mongourl = `${this.config.protocol}://`
+    if (this.config.username) mongourl += `${this.config.username}${this.config.password ? `:${this.config.password}` : ''}@`
+    mongourl += `${this.config.host}${this.config.port ? `:${this.config.port}` : ''}/${this.config.authDatabase ? this.config.authDatabase : this.config.name}`
+    if (this.config.connectionOptions) {
```

Comment:
what about `config.params`

Reply:
https://docs.mongodb.com/manual/reference/connection-string/
connection string refers to the "params" as "options", and options.options/ config.options feels confusing to me, thus why I chosed connectionOptions.
I also prefer a shorter key, so I am not sure which to use.